activation/validation.go

```diff
@@ -63,8 +63,8 @@ func (v *Validator) NIPost(ctx context.Context, publishEpoch types.EpochID, node
 		return 0, err
 	}
 
-	if err := v.Post(ctx, publishEpoch, nodeId, commitmentAtxId, nipost.Post, nipost.PostMetadata, numUnits, opts...); err != nil {
-		return 0, fmt.Errorf("invalid Post: %w", err)
+	if err := v.Post(ctx, nodeId, commitmentAtxId, nipost.Post, nipost.PostMetadata, numUnits, opts...); err != nil {
+		return 0, fmt.Errorf("invalid Post: %v", err)
```

Comment:
Why replace `%w` with `%v`? Shouldn't `%w` be used to signal the source of the error when bubbling up?

Reply:
oversight